src/enums/partyTypes 2.ts

```diff
@@ -0,0 +1,4 @@
+export enum PartyTypes {
+  PERSON = 'person',
+  ORGANIZATION = 'organization'
+}
```

Comment:
Why does this filename (and orgPersonTypes) contain a "2" in it?

Reply:
Good question, my editor pulling some tricks i think. Will remove this file as it was most likely duplicated. This sometimes happens.

Reply:
Removed these duplicate files 😵 
Still not sure how this sometimes happens. 100% an editor config